Automattic/wp-calypso
# Launchpad: Add launchpad sidebar upsell domain task unit tests

Related to #73777

## Test/Review Estimation

Test: short
Review: short

## Proposed Changes

* Adds unit tests for the Launchpad domain upsell task

## Testing Instructions



* Checkout this branch
* Confirm the Launchpad unit tests are passing by running `yarn run test-client client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/` 

## Pre-merge Checklist
